templates/vue/src/utils/test.js

```diff
@@ -0,0 +1,40 @@
+import { render as r } from "@testing-library/vue"
+import { store } from "@/store"
+import { parse } from "./dataset"
+
+const settings = {
+  tapestrySlug: "testing",
```

Comment:
What's a slug and when should it be used?

Reply:
All tapestries have a slug, just like any other Wordpress post. [https://www.wpbeginner.com/glossary/post-slug/](https://www.wpbeginner.com/glossary/post-slug/)